src/ofApp.cpp

```diff
@@ -16,41 +17,84 @@ void ofDisplayApp::setup() {
 
 	piMapper.registerFboSource(covid19);
 	piMapper.setup();
+
+	loadCovidCsv();
+
+	ofLog() << "Loaded country . " << covidData[0].countryRegion;
+	ofLog() << "Loaded country . " << covidData[1].countryRegion;
 }
 
-void ofDisplayApp::update() {
-	piMapper.update();
+void ofDisplayApp::loadCovidCsv() {
```

Comment:
Should this just be it's own class that handles loading it and making it available?

Reply:
That'd be a sensible change.